apps/docs/spec/supabase_dart_v2.yml

```diff
@@ -659,6 +691,88 @@ functions:
+        isSpotlight: true
+        code: |
+          ```dart
+          final identities = await supabase.auth.getUserIdentities();
+          ```
+  - id: link-identity
+    title: 'linkIdentity()'
+    description: |
+      Links an oauth identity to an existing user. This method supports the PKCE flow.
+    notes: |
+      - The **Enable Manual Linking** option must be enabled from your [project's authentication settings](/dashboard/project/_/settings/auth).
+      - The user needs to be signed in to call `linkIdentity()`.
+      - If the candidate identity is already linked to the existing user or another user, `linkIdentity()` will fail.
+    params:
+      - name: provider
+        isOptional: false
+        type: OAuthProvider
+        description: The provider to link the identity to.
+      - name: redirectTo
+        isOptional: true
+        type: String
+        description: The URL to redirect the user to after they sign in with the third-party provider.
+      - name: scopes
+        isOptional: true
+        type: String
+        description: A list of scopes to request from the third-party provider.
+      - name: authScreenLaunchMode
+        isOptional: true
+        type: LaunchMode
+        description: The launch mode for the auth screen. Defaults to `LaunchMode.platformDefault`.
+      - name: queryParams
+        isOptional: true
+        type: Map<String, String>
+        description: Additional query parameters to be passed to the OAuth flow.
+    examples:
+      - id: link-identity
+        name: Link an identity to a user
+        isSpotlight: true
+        code: |
+          ```dart
+          await supabase.auth.linkIdentity(OAuthProvider.google);
+          ```
+  - id: unlink-identity
+    title: 'unlinkIdentity()'
+    description: |
+      Unlinks an identity from a user by deleting it. The user will no longer be able to sign in with that identity once it's unlinked.
+    params:
+      - name: identity
+        isOptional: false
+        type: UserIdentity
+        description: The user identity to unlink.
+    notes: |
+      - The **Enable Manual Linking** option must be enabled from your [project's authentication settings](/dashboard/project/_/settings/auth).
+      - The user needs to be signed in to call `unlinkIdentity()`.
+      - The user must have at least 2 identities in order to unlink an identity.
+      - The identity to be unlinked must belong to the user.
+    examples:
+      - id: unlink-identity
+        name: Unlink an identity
+        isSpotlight: true
+        code: |
+          ```dart
+          // retrieve all identites linked to a user
+          final identities = await supabase.auth.getUserIdentities();
+
+          // find the google identity 
+          final googleIdentity = identities.firstWhere(
+            (element) => element.provider == 'google',
+          );
+
+          // unlink the google identity
+          await supabase.auth.unlinkIdentity(identities.first);
```

Comment:
should this be `googleIdentity`?

Reply:
Oops, thanks for catching this. Will push a fix.